### ISSUE TITLE
Crash to Desktop with latest Pre-Release

### ISSUE THREAD

**Issue by PFCKrutonium** [CorsixTH/CorsixTH#815]
Invalid Theme Hospital folder specified in config file, as at least the following files are missing: Data\VBlk-0.tab, Levels\Level.L1, QData\SPointer.dat.
Please make sure that you point the game to a valid copy of the data files from the original game, as said files are required for graphics and sounds.
Trying to let the user select a new one.
An error has occured!
Almost anything can be the cause, but the detailed information below can help the developers find the source of the error.
Running: The frame handler.
A stack trace is included below, and the handler has been disconnected.
C:\Program Files\CorsixTH\Lua\window.lua:403: bad argument #3 to 'drawRect' (number has no integer representation)
stack traceback:
        [C]: in method 'drawRect'
        C:\Program Files\CorsixTH\Lua\window.lua:403: in method 'custom_draw'
        C:\Program Files\CorsixTH\Lua\window.lua:1261: in field 'draw'
        C:\Program Files\CorsixTH\Lua\dialogs\tree_ctrl.lua:660: in method 'draw'
        C:\Program Files\CorsixTH\Lua\window.lua:1277: in field 'draw'
        C:\Program Files\CorsixTH\Lua\dialogs\resizable.lua:111: in field 'draw'
        ...es\CorsixTH\Lua\dialogs\resizables\directory_browser.lua:201: in method 'draw'
        C:\Program Files\CorsixTH\Lua\window.lua:1277: in field 'draw'
        C:\Program Files\CorsixTH\Lua\ui.lua:324: in method 'draw'
        C:\Program Files\CorsixTH\Lua\app.lua:897: in function <C:\Program Files\CorsixTH\Lua\app.lua:890>
        (...tail calls...)
        C:\Program Files\CorsixTH\Lua\app.lua:779: in function <C:\Program Files\CorsixTH\Lua\app.lua:774>

Warning: No event handler for frame

This happens when attempting to open 
C:\Program Files (x86)\Origin Games\, 

with the game being stored at 
C:\Program Files (x86)\Origin Games\Theme Hospital\data\Game

On the screen where you select where your game files are.

**Comment by bobcanthelpyou** [comment]
Do you use the v0.50 beta 2 release? or one of the v0.40 rc releases?

Could you reproduce the error when using
```
C:\Program Files (x86)\Origin Games\Theme Hospital\
```
as Theme Hospital Install folder?

```
-- Theme hospital install folder: original game data files are loaded from this
-- folder. Between the square braces should be the folder which contains the
-- original HOSPITAL.EXE and/or HOSP95.EXE file. This can point to a copy of
-- the Theme Hospital demo, though a full install of the original game is
-- preferred.
```

**Comment by PFCKrutonium** [comment]
0.50 b2, And if I manually specify the folder in the config it works fine, but I can't navigate to it via the UI.

**Comment by TheCycoONE** [comment]
This issue was fixed already, after the release of 0.50 beta 2 in 65c85b94f0db
